tests/saitests/sai_qos_tests.py

```diff
@@ -2134,6 +2135,133 @@ def runTest(self):
         finally:
             sai_thrift_port_tx_enable(self.client, asic_type, [dst_port_id])
 
+class LossyQueueVoqTest(sai_base_test.ThriftInterfaceDataPlane):
+    def runTest(self):
+        switch_init(self.client)
+
+        # Parse input parameters
+        dscp = int(self.test_params['dscp'])
+        ecn = int(self.test_params['ecn'])
+        pg = int(self.test_params['pg']) + 2 # The pfc counter index starts from index 2 in sai_thrift_read_port_counters
+        sonic_version = self.test_params['sonic_version']
+        router_mac = self.test_params['router_mac']
+        dst_port_id = int(self.test_params['dst_port_id'])
+        dst_port_ip = self.test_params['dst_port_ip']
+        dst_port_mac = self.dataplane.get_mac(0, dst_port_id)
+        src_port_id = int(self.test_params['src_port_id'])
+        src_port_ip = self.test_params['src_port_ip']
+        src_port_mac = self.dataplane.get_mac(0, src_port_id)
+        asic_type = self.test_params['sonic_asic_type']
+        hwsku = self.test_params['hwsku']
+
+        # get counter names to query
+        ingress_counters, egress_counters = get_counter_names(sonic_version)
+
+        # prepare tcp packet data
+        ttl = 64
+
+        pkts_num_leak_out = int(self.test_params['pkts_num_leak_out'])
+        pkts_num_trig_egr_drp = int(self.test_params['pkts_num_trig_egr_drp'])
+        if 'packet_size' in self.test_params.keys():
+            packet_length = int(self.test_params['packet_size'])
+            cell_size = int(self.test_params['cell_size'])
+            if packet_length != 64:
+                cell_occupancy = (packet_length + cell_size - 1) / cell_size
+                pkts_num_trig_egr_drp /= cell_occupancy
+        else:
+            packet_length = 64
+
+        pkt_dst_mac = router_mac if router_mac != '' else dst_port_mac
+        # crafting 2 udp packets with different udp_dport in order for traffic to go through different flows
+        pkt = simple_udp_packet(pktlen=packet_length,
```

Comment:
As the voq mode is default, could you indicate the reason why we need go through the different flows using two different udp pkt?

Reply:
We have kept 2 flows and reduced the number of packets to half in default voq case to keep the testcase uniform . Both flows will go on same voq .